.github/workflows/docs.yaml

```diff
@@ -0,0 +1,32 @@
+name: Documentation
+# This workflow builds the static documentation site, and publishes the results to GitHub Pages.
+# It runs on every push to the main branch, with changes in the docs and spec directories
+on:
+  workflow_dispatch:  # allow manual updates
+  push:
+    branches:
+      - main
+    paths:
+      - "docs/**"
+      - "spec/**"
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    container:
+      image: tendermintdev/docker-website-deployment
```

Comment:
Where does the `Dockerfile` for this image live?

Reply:
https://github.com/tendermint/images/blob/master/docker-website-deployment/Dockerfile

I have it on my list to move this to ghcr.io, but I believe cosmos also uses it so some care may be required.

Reply:
See also: https://github.com/tendermint/tendermint/issues/7297